app/backend/app.py

```diff
@@ -133,36 +138,31 @@
 model_name = ''
 model_version = ''
 
-if str_to_bool.get(ENV["IS_GOV_CLOUD_DEPLOYMENT"]):
-    model_name = ENV["AZURE_OPENAI_CHATGPT_MODEL_NAME"]
-    model_version = ENV["AZURE_OPENAI_CHATGPT_MODEL_VERSION"]
-    embedding_model_name = ENV["AZURE_OPENAI_EMBEDDINGS_MODEL_NAME"]
-    embedding_model_version = ENV["AZURE_OPENAI_EMBEDDINGS_VERSION"]
-else:
-    # Set up OpenAI management client
-    openai_mgmt_client = CognitiveServicesManagementClient(
-        credential=azure_credential,
-        subscription_id=ENV["AZURE_SUBSCRIPTION_ID"])
+# Set up OpenAI management client
+openai_mgmt_client = CognitiveServicesManagementClient(
+    credential=azure_credential,
+    subscription_id=ENV["AZURE_SUBSCRIPTION_ID"],
+    base_url=ENV["AZURE_ENDPOINTS_MANAGEMENT_API"])
```

Comment:
This is a breaking change. See issue #517 and corresponding PR

Reply:
This was using the wrong endpoint, thus the issue. This param is required for this call to work. I have updated the BICEP deployment to populate with the proper value for AzureCloud and AzureUSGovernment now and it matches per the documentation default values at https://learn.microsoft.com/en-us/python/api/azure-mgmt-cognitiveservices/azure.mgmt.cognitiveservices.cognitiveservicesmanagementclient?view=azure-python#parameters

Reply:
"Roger, Roger" I think the one that was failing had core prepended while this one doesn't. Makes sense then, apologies.